packages/shared/src/hooks/squads/usePostToSquad.ts

```diff
@@ -67,25 +89,83 @@ export const usePostToSquad = ({
   const { completeAction } = useActions();
   const [preview, setPreview] = useState(initialPreview);
   const { requestMethod } = useRequestProtocol();
+
+  const {
+    mutateAsync: onCreatePost,
+    isPending: isLoadingFreeform,
```

Comment:
Ideally we shouldn't recast to loading to avoid confusion between devs.
But non blocking